plugins/woocommerce/changelog/add-53194-gateway-recommended-payment-methods

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: Expose payment gateway recommended payment methods through the providers API.
```

Comment:
Nitpick: We can improve this slightly as:

```suggestion
Comment: Expose the recommended payment methods for payment gateways (currently only WooPayments) through the Providers API.
```

Reply:
Thank you, @anu-rock! Words are important!